node/src/types/deploy.rs

```diff
@@ -295,9 +317,28 @@ pub struct Deploy {
 }
 
 impl Deploy {
-    /// Constructs a new `Deploy`.
+    /// Constructs a new unsigned `Deploy`
     #[allow(clippy::too_many_arguments)]
     pub fn new(
+        hash: DeployHash,
+        header: DeployHeader,
+        payment: ExecutableDeployItem,
+        session: ExecutableDeployItem,
+        approvals: Vec<Approval>,
+    ) -> Self {
```

Comment:
Let's revert this change since it seems like this `new()` constructor isn't used anywhere.

Reply:
Will do